lib/editor/components/EntityDetailsHeader.js

```diff
@@ -133,6 +139,11 @@ export default class EntityDetailsHeader extends Component<Props> {
     const zoomDisabled = activeEntity && !subComponent
       ? mapState.target === activeEntity.id
       : mapState.target === subEntity
+    // prevent zooming to empty trip pattern
+    const emptyTripPattern = activeEntity.tripPatterns && activeEntity.tripPatterns.some(pattern => pattern.patternLocations.length === 0 && pattern.patternStops.length === 0)
+    // flex zoom available only when a trip pattern is actually rendered on the map
+    const tripPatternRendered = activeEntity.tripPatterns && !(this.props.params ? this.props.params.subEntityId : false)
```

Comment:
The extra negation is confusing! I'm not sure if this feels cleaner though... What do you think
```suggestion
    const tripPatternRendered = activeEntity.tripPatterns && (this.props.params ? !this.props.params.subEntityId : true)
```